src/plugins/LADTable/LADTableViewProvider.js

```diff
@@ -43,6 +47,6 @@ export default class LADTableViewProvider {
     }
 
     priority(domainObject) {
-        return 1;
+        return this.openmct.priority.HIGH;
```

Comment:
Priority set to high since previously the LAD Table and Telemetry Table had an equal priority of 1.